flink-runtime/src/main/java/org/apache/flink/runtime/state/AbstractStateBackend.java

```diff
@@ -66,4 +66,9 @@ public OperatorStateBackend createOperatorStateBackend(
 			String operatorIdentifier) throws Exception {
 		return new DefaultOperatorStateBackend(env.getUserClassLoader());
 	}
+
+	@Override
+	public void close() throws Exception {
+
+	}
```

Comment:
I absolutely agree, but I am afraid that this might be a breaking API change. Maybe @aljoscha can give a second opinion about this.